gui/mozregui/report.py

```diff
@@ -60,11 +61,33 @@ def status_text(self):
         return msg % self.state_text
 
 
+def bulk_action_slots(slots, signal_object, slot_object, action):
```

Comment:
hmm, I would put the action argument at first. This way you could remove bulk_connect_slots and bulk_disconnect_slots:
```
bulk_action_slots('connect', slots, signal_object, slot_object)
bulk_action_slots('disconnect', slots, signal_object, slot_object)
```
This is more readable to me. Or you can keep the order you used, but I don't think the two others functions are meaningful.